configs/components/cfpropertylist.rb

```diff
@@ -1,7 +1,7 @@
 component "cfpropertylist" do |pkg, settings, platform|
   pkg.version "2.2.7"
   pkg.md5sum "be2fc619084422cc28cb68f6084bea91"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/CFPropertyList-2.2.7.tar.gz"
+  pkg.url "https://rubygems.org/downloads/CFPropertyList-#{pkg.get_version}.gem"
```

Comment:
The tarball URL looks like this: `https://github.com/ckruse/CFPropertyList/archive/cfpropertylist-#{pkg.get_version}.tar.gz`

Note that it's lowercase. The tarball internally is not, for one reason or another; that, coupled with GitHub's 302s on 302s on 302s, is how I found the "Vanagon doesn't handle redirects" problem.